app/src/main/java/com/severianfw/picto/view/home/HomeFragment.kt

```diff
@@ -78,6 +81,13 @@ class HomeFragment : Fragment() {
     private fun setupPhotoRecyclerView() {
         val gridLayoutManager = GridLayoutManager(context, SPAN_COUNT)
         val photoAdapter = PhotoAdapter()
+        photoAdapter.setOnItemClickListener(object : PhotoAdapter.OnItemClickListener {
+            override fun onItemClick(photo: PhotoItemModel) {
+                val detailIntent = Intent(activity, DetailActivity::class.java)
+                detailIntent.putExtra(DetailActivity.TAG, photo)
```

Comment:
I think we can make a Constant.kt 
```
const val PHOTO_ITEM = "PHOTO_ITEM"
```

Then we can remove `TAG` of `DetailActivity`